lambdas/tests/unit/services/session-request-validator.test.ts

```diff
@@ -297,6 +292,167 @@ describe("session-request-validator.ts", () => {
         });
     });
 
+    describe("sessionRequestValidator for di-ipv-cri-kbv-api", () => {
+        let sessionRequestValidator: SessionRequestValidator;
+        let sessionRequestValidationConfig: SessionRequestValidationConfig;
+        const jwtVerifier = jest.mocked(JwtVerifier);
+
+        beforeEach(() => {
+            sessionRequestValidationConfig = {
+                expectedJwtRedirectUri: "redirect-uri",
+            } as SessionRequestValidationConfig;
+
+            sessionRequestValidator = new SessionRequestValidator(
+                sessionRequestValidationConfig,
+                jwtVerifier.prototype,
+                { verificationScore: [1, 2] } as CRIEvidenceProperties,
+            );
+        });
+
+        it("should pass when verificationScore is 1 and cri is di-ipv-cri-kbv-api", async () => {
```

Comment:
Why is the CRI important here? Because its needed for the SSM param path to compare the allowed evidence requested values against the evidence requested included in the session request?

Reply:
Good spot, it was just from C&Ping other tests. I have removed it from these tests.